week-1/E-strings/exercise.js

```diff
@@ -1,3 +1,5 @@
 // Start by creating a variable `message`
-
-console.log(message);
+var message="this is a string";
+var messageType=typeof message;
+console.log(messageType); 
+console.log(messageType);
```

Comment:
Your code is good but the expected result for this exercise is to log the message and the type, you're consol logging the type twice